mimesis/providers/internet.py

```diff
@@ -37,6 +39,11 @@ def __init__(self, *args, **kwargs):
         """
         super().__init__(*args, **kwargs)
         self.__file = File(seed=self.seed)
+        # host names consist of syntax-restricted labels
+        # https://tools.ietf.org/html/rfc8499#page-8
+        self.__LDHs = [w for w in Text('en')._data['words']['normal']
+                       if len(w) <= 63 and re.fullmatch(
```

Comment:
This might be a bit expensive to run at every initialization, but it's more predictable than lazy-loading the list upon first use of `hostname` and more convenient to derive from English words than from a new list. I'm happy to change it upon request.